app/index.js

```diff
@@ -132,6 +132,12 @@ module.exports = class extends Generator {
       applicationType: this.options.spec && this.options.spec.applicationType
     });
 
+    if(this.options.spec && (this.options.spec.applicationType === 'MS' || this.options.spec.applicationType === 'BLANK')) {
+      this._writeHandlebarsFile('server/routers/health.js', 'server/routers/health.js', {
+        appType: this.options.spec.applicationType
```

Comment:
the parameter here is called `appType`, but you send in `applicationType` in line 132. Is that right?

Reply:
Ah yeah thanks for catching that

Reply:
This is what I'm referencing https://github.ibm.com/arf/starter-microservice-basic/blob/development/blueprint.json#L87